cli/bin/ci/run-integration-tests.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+# Usage: run-integration-tests.sh
+#
+# Runs the Waiter CLI integration tests using the (local) shell scheduler.
+
+set -e
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+CLI_DIR=${DIR}/../..
+ROOT_DIR=${CLI_DIR}/..
+
+export GRAPHITE_SERVER_PORT=5555
```

Comment:
Why does the cli need to start a graphite server?

Reply:
That's a good question.

I believe @nsinkov is working on fixing this so that it's not needed. @nsinkov, can you confirm?

Reply:
@shamsimam @dposada using bin/run-using-shell-scheduler.sh causes waiter to pull in a setting that expects a graphite server. need to refactor bin/run-using-shell-scheduler.sh we have a ticket open for that

Reply:
@shamsimam Does this work for you?

Reply:
Yes.